tools/ports/libjpeg.py

```diff
@@ -16,7 +16,10 @@ def needed(settings):
 
 
 def get(ports, settings, shared):
-  ports.fetch_project('libjpeg', 'https://www.ijg.org/files/jpegsrc.v9c.tar.gz', 'jpeg-9c', sha512hash=HASH)
+  # Archive mirrored from http://www.ijg.org/files/jpegsrc.v9c.tar.gz.
+  # We have issues where python urllib was not able to load from the www.ijg.org webserver
+  # and was resulting in 403: Forbidden.
+  ports.fetch_project('libjpeg', 'https://storage.googleapis.com/webassembly/emscripten-ports/jpegsrc.v9c.tar.gz', 'jpeg-9c', sha512hash=HASH)
```

Comment:
seems fine if this was easier for you. another option would be to open a repo on emscripten-ports.

Reply:
I was thinking for pure-mirros it would make more sense to use cloud storage.   In the past we have had issues with github archives changing their hashes (although admitently that is only for the "auto-archive" thing that github provides).    It also seems like overkill to create N repos just to mirror N zip file.